deno_core looks like this:

    [features]
    default = ["v8_use_custom_libcxx"]
    v8_use_custom_libcxx = ["v8/use_custom_libcxx"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -43 +43 @@
-deno_core = { version = "0.229.0" }
+deno_core = { version = "0.229.0", default-features = false }
